extension/model/static/errorReporting/sentry.js

```diff
@@ -5694,35 +5668,6 @@ var Sentry = (function (exports) {
         catch (_oO) { }
         return sentryWrapped;
     }
-    /**
-     * Injects the Report Dialog script
-     * @hidden
-     */
-    function injectReportDialog(options) {
-        if (options === void 0) { options = {}; }
-        if (!global$2.document) {
-            return;
-        }
-        if (!options.eventId) {
-            logger.error('Missing eventId option in showReportDialog call');
-            return;
-        }
-        if (!options.dsn) {
-            logger.error('Missing dsn option in showReportDialog call');
-            return;
-        }
-        var script = global$2.document.createElement('script');
-        script.async = true;
-        script.src = getReportDialogEndpoint(options.dsn, options);
-        if (options.onLoad) {
-            // eslint-disable-next-line @typescript-eslint/unbound-method
-            script.onload = options.onLoad;
-        }
-        var injectionPoint = global$2.document.head || global$2.document.body;
-        if (injectionPoint) {
-            injectionPoint.appendChild(script);
-        }
-    }

```

Comment:
I can't get github to add a comment in context, so putting this here. Down in `UserAgent.prototype.setupOnce`, it seems an URL is collected. Can you check and scrub this URL as well?

Reply:
This method saves url to `event.request.url` and I already scrub it

Reply:
I decided to delete `event.request.url` completely, it wasn't very useful anyway